pkg/kubelet/apis/config/validation/validation.go

```diff
@@ -200,9 +200,9 @@ func ValidateKubeletConfiguration(kc *kubeletconfig.KubeletConfiguration) error
 	}
 	allErrors = append(allErrors, metrics.ValidateShowHiddenMetricsVersion(kc.ShowHiddenMetricsForVersion)...)
 
-	logOption := logs.NewOptions()
+	logOption := logsoptions.NewOptions()
 	if kc.Logging.Format != "" {
-		logOption.LogFormat = kc.Logging.Format
+		logOption.Config.Format = kc.Logging.Format
```

Comment:
This was a workaround for defaulting, we can skip using NewOptions and instead call `ValidateLoggingConfiguration` directly on `kc.Logging`